src/journal.sh

```diff
@@ -266,6 +266,13 @@ rlJournalEnd(){
         [[ "$BEAKERLIB_JOURNAL" == "0" ]] || rlLog "JOURNAL XML: $__INTERNAL_BEAKERLIB_JOURNAL"
         rlLog "JOURNAL TXT: $__INTERNAL_BEAKERLIB_JOURNAL_TXT"
     fi
+    if (( ${#rlRun_LOG[@]} ))
+    then
+    rm -f "${rlRun_LOG[@]}"
+    rlLogInfo "Deleting rlRun_LOG file(s)"
+    local tmp_LOG=${rlRun_LOG[*]}
+    rlLogDebug "Deleting temporary file(s) ${tmp_LOG// /, }"
+    fi
```

Comment:
nicer and consistent code formating
```suggestion
    if (( ${#rlRun_LOG[@]} )); then
        rm -f "${rlRun_LOG[@]}"
        rlLogInfo "Deleting rlRun_LOG file(s)"
        local tmp_LOG=${rlRun_LOG[*]}
        rlLogDebug "Deleting temporary file(s) ${tmp_LOG// /, }"
    fi
```

Reply:
Could you also update the documentation of `rlRun` to inform user about the deletion of the files in `rlJournalEnd`?